### DISCUSSION TITLE
Option to log out from Keycloak when logging out of Data Studio App

### DISCUSSION THREAD

**Discussion by jofmi** [directus/directus#24699] [Feature Requests]
### Describe the Improvement

We use Seamless SSO with Keycloak and there is often confusion when users log in to Keycloak through the Data Studio because when they log out, they remain logged in with Keycloak.

It would be great to have a boolean config variable such as "AUTH_KEYCLOAK_LOGOUT_FLOW".

If set to true, logging out of directus studio should log out the user from Keycloak as well.

This could either be done through the Keycloak API or by redirecting to `<KEYCLOAK_URL>/realms/<KEYCLOAK_REALM>/protocol/openid-connect/logout` after logging out of Directus.

**Comment by vanling** [comment]
I would love the same for other (custom) identity providers. Some sort of hook/flow action `auth.logout`?

**Comment by alessioVelluso** [comment]
Did this became a thing? Would really love to have that option too
